src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -148,47 +161,59 @@ const CrossPodWriteForm = () => {
-            id="cross-upload-doc"
-            size="60"
-            name="crossPodUpload"
-            {...username}
-            placeholder={selectedUser}
-          />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc">Select document type to upload: </label>
-          <DocumentSelection htmlId="upload-doc" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
-          <br />
-          <br />
-          <textarea id="upload-doc-desc" name="description" {...description} />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doctype">File to upload: </label>
-          <input
-            id="upload-doctype"
-            type="file"
-            name="uploadDoctype"
-            accept=".pdf, .docx, .doc, .txt, .rtf"
-            onChange={handleFileChange}
-          />
-          <button disabled={state.processing} type="submit">
-            Upload file
-          </button>
-        </div>
-      </form>
+      <Box sx={{ minWidth: 120 }}>
+        <form onSubmit={handleCrossPodUpload} autoComplete="off">
+          <div style={formRowStyle}>
+            <label htmlFor="cross-upload-doc">Search document from username: </label>
+            <br />
+            <br />
+            <input
+              id="cross-upload-doc"
+              size="60"
+              name="crossPodUpload"
+              {...username}
+              placeholder={selectedUser}
+            />
+          </div>
+          <div style={formRowStyle}>
+            <FormControl fullWidth autoComplete="off">
+              <InputLabel id="upload-doc">
+                <i>Select Document Type</i>
+              </InputLabel>
+              <DocumentSelection htmlId="upload-doc" value={documentType} onChange={handleChange} />
+            </FormControl>
+          </div>{' '}
+          <div style={formRowStyle}>
+            <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
+            <input id="upload-doc-expiration" name="date" type="date" />
+          </div>
+          <div style={formRowStyle}>
+            <label htmlFor="upload-doc-desc">Enter description: </label>
+            <br />
+            <br />
+            <textarea id="upload-doc-desc" name="description" {...description} />
+          </div>
+          <div style={formRowStyle}>
+            <label htmlFor="upload-doctype">File to upload: </label>
+            <input
+              id="upload-doctype"
+              type="file"
+              name="uploadDoctype"
+              accept=".pdf, .docx, .doc, .txt, .rtf"
+              onChange={handleFileChange}
+            />
+            <Button
```

Comment:
This button should be outside of `<FormControl>` following FetchDocumentsForm.jsx